apis/supernova/src/qae.js

```diff
@@ -66,8 +66,9 @@ const resolveValue = (data, reference, defaultValue) => {
 function target(def) {
   const propertyPath = def.path || '/qHyperCubeDef';
   const layoutPath = propertyPath.slice(0, -3);
-  if (/\/qHyperCube$/.test(layoutPath) === false) {
-    throw new Error(`Incorrect definition for qHyperCubeDef at ${propertyPath}`);
+  if (/\/(qHyperCube|qListObject)$/.test(layoutPath) === false) {
+    const d = layoutPath.includes('/qHyperCube') ? 'qHyperCubeDef' : 'qListObjectDef';
+    throw new Error(`Incorrect definition for ${d} at ${propertyPath}`);
```

Comment:
Think we should spell this out a bit more. Simply getting `Incorrect ndeinition for qHyperCubeDef at myProps.qHyperCubeDef` for example doesn't really help anyone.
Add another sentence with: `Valid paths include /qHyperCubeDef or /qListObject, eg data/qHyperCubeDef`

Reply:
Updated!